main.go

```diff
@@ -68,14 +70,14 @@ func run() error {
 								Usage:     "generate OpenAPI JSON from JSight",
```

Comment:
It might be better to write: converts JSight to OpenAPI JSON

Reply:
fixed